docroot/themes/humsci/humsci_basic/src/js/tables/wrap.js

```diff
@@ -18,9 +18,14 @@ function wrapElement(element) {
 
 // Select every table element
 const elements = document.querySelectorAll('table');
+const uiPatternTable = document.querySelectorAll('.hb-table-pattern');
```

Comment:
We shouldn't we need to wrap the table pattern like we do table elements. This class is only to make the table have a horizontal scroll.

Reply:
On pages with the sidebar navigation and a 5 column table, there is a small window of screen sizes where the 5th column in the table creates horizontal page scroll. This isn't an issue when a page doesn't have the secondary nav, such as the Econ Courses page.